airsonic-main/src/test/java/org/airsonic/player/service/InternetRadioServiceTest.java

```diff
@@ -80,15 +80,11 @@ public void setup() throws Exception {
 
         // Prepare the mocked URL connection for the redirection to simple playlist
         HttpURLConnection mockURLConnectionMove = Mockito.mock(HttpURLConnection.class);
-        InputStream mockURLInputStreamMove = new ByteArrayInputStream("".getBytes());
-        doReturn(mockURLInputStreamMove).when(mockURLConnectionMove).getInputStream();
```

Comment:
Spring Boot 2 will error here because the new Mockito framework does not have lenient mode on by default to prevent excessive stubbing